efficientdet/keras/utils_keras.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Common utils."""
+
+# gtype import
+from __future__ import absolute_import, division, print_function
+
+from typing import Text, Union
+
+import tensorflow as tf
+
+# pylint: disable=logging-format-interpolation
+from utils import BatchNormalization, TpuBatchNormalization
+
+
+class ActivationFn(tf.keras.layers.Layer):
+  def __init__(self, act_type: Text, name='activation_fn', **kwargs):
+
+    super(ActivationFn, self).__init__()
+
+    self.act_type = act_type
+
+    if act_type == 'swish':
+      self.act = tf.nn.swish
+    elif act_type == 'swish_native':
+      self.act = lambda x: x * tf.sigmoid(x)
+    elif act_type == 'relu':
+      self.act = tf.nn.relu
+    elif act_type == 'relu6':
+      self.act = tf.nn.relu6
+    else:
+      raise ValueError('Unsupported act_type {}'.format(act_type))
+
+    self.layer = tf.keras.layers.Lambda(lambda x: self.act(x), name=name)
```

Comment:
Remove Lambda.
Over wrapper make developers annoyed.

Reply:
My bad it was just for the naming tests, it have no purpose, thx for pointing it out